src/notifier.js

```diff
@@ -708,15 +699,22 @@ NotifierPrototype.uncaughtError = _wrapNotifierFn(function(message, url, lineNo,
     'useragent': navigator.userAgent
   };
 
-  var payload = this._buildPayload(new Date(), this.options.uncaughtErrorLevel, message, stack);
-  this._enqueuePayload(payload, true, [this.options.uncaughtErrorLevel, message, url, lineNo, colNo, err]);
+  var payload = this._buildPayload(new Date(), this.options.uncaughtErrorLevel,
+    message, stack, context);
+  this._enqueuePayload(payload, true, [this.options.uncaughtErrorLevel,
+    message, url, lineNo, colNo, err]);
 });
 
 
 NotifierPrototype.global = _wrapNotifierFn(function(options) {
   options = options || {};
+  var knownOptions = {
+    startTime: options.startTime,
+    maxItems: options.maxItems,
+    itemsPerMinute: options.itemsPerMinute
+  };
 
-  Util.merge(window._globalRollbarOptions, options);
+  extend(true, window._globalRollbarOptions, knownOptions);
```

Comment:
We were giving the user the ability to add *any* keys to the global options. This change limits which keys will be put into the global options object based on which ones we support. With this change, if the user passes in an option we don't support, we will silently not put it into the global options object.